src/modules/watch/router.js

```diff
@@ -25,9 +25,23 @@ module.exports = async (server, opts) => {
     }
   })
 
-  server.put('/:id/targets', { schema: schema.updateTargets }, async (req, res) => {
+  server.put(
+    '/targets/:id/data',
+    { schema: schema.updateTargetData },
+    async (req, res) => {
+      try {
+        await watchService.updateTarget(req.params.id, req.body.data)
+        res.code(204).send()
+      } catch (err) {
+        req.log.error(err.message)
+        res.code(500).send()
+      }
+    }
+  )
+
+  server.put('/:id/check', { schema: schema.updateLastCheck }, async (req, res) => {
```

Comment:
```suggestion
  server.put('/:id/checkedAt', { schema: schema.updateCheckedAt }, async (req, res) => {
```
Db lưu thế nào thì ông viết lại thế thôi là được mà.